content/concept-transaction-cost.md

```diff
@@ -7,24 +7,74 @@ Every transaction must [specify how much XRP it will destroy](#specifying-the-tr
 
 ## Current Transaction Cost ##
 
-The current transaction cost required by the network is typically **0.01 XRP** (10,000 drops), although it sometimes increases due to network load.
+The current transaction cost required by the network for a standard transaction is typically **0.01 XRP** (10,000 drops), although it sometimes increases due to network load.
 
 You can also [query `rippled` for the current transaction cost](#querying-the-transaction-cost).
 
+### Special Transaction Costs ###
+
+Some transactions have different transaction costs:
+
+| Transaction           | Cost Before Load Scaling |
+|-----------------------|--------------------------|
+| Reference Transaction (Standard cost of most transactions) | 10 drops |
+| [Key Reset Transaction](#key-reset-transaction) | 0 |
+| [Multi-signed transaction](reference-transaction-format.html#multi-signing) | 10 drops × (1 + Number of Signatures Provided) |
+
 
 ## Beneficiaries of the Transaction Cost ##
 
 The transaction cost is not paid to any party: the XRP is irrevocably destroyed. Since no new XRP can ever be created, this makes XRP more scarce, and consequently benefits all holders of XRP by making XRP more valuable.
 
 
-## Load Scaling ##
+## Load Cost and Open Ledger Cost ##
+
+When the [FeeEscalation amendment](concept-amendments.html#feeescalation) is enabled, there are two thresholds for the transaction cost:
```

Comment:
I played around with a number of rephrasings and I think all of them are less clear than the current one.

Part of the reason I'm leaving it as-is has to do with parallel ledger chains, such as the ones created in stand-alone mode from a fresh genesis ledger. Those ledgers don't have amendments enabled (unless you force them on with the `[feature]` config stanza) so it's not just a matter of "when" the amendment gets enabled, but "in situations where the amendment is enabled". I think "When" with present tense is the most efficient way to explain the situation without introducing a large amount of additional information that may also need to be updated. It's like saying, "When on vacation, there are..."